src/app/shared/components/header/header.component.html

```diff
@@ -1,15 +1,17 @@
 <mat-toolbar class="c-header">
   <mat-toolbar-row>
-    <sofie-logo [text]="title" (click)="navigateHome()"></sofie-logo>
+    <a class="logo-link" [routerLink]="[Paths.HOME]">
```

Comment:
I am unsure whether this should start with `'/', `?
It seems to work without. Not sure why :)

Reply:
It's work, and is not work with `'/'`